packages/pg-v5/commands/copy.js

```diff
@@ -21,13 +21,12 @@ async function run(context, heroku) {
   let resolve = async function (db) {
     if (db.match(/^postgres:\/\//)) {
       // For the case an input is URL format
-      let uri = url.parse(db)
-      let dbname = uri.path ? uri.path.slice(1) : ''
-      let host = `${uri.hostname}:${uri.port || 5432}`
+      let conn = util.parsePostgresConnectionString(db)
+      let host = `${conn.host}:${conn.port}`
```

Comment:
The one difference I notice is the loss of the default port (5432) if one is not set, just want to confirm that that's ok.

Reply:
Good eye, @tevanoff the parser defaults it by itself to 5432: https://github.com/heroku/cli/pull/1882/files#diff-1c49bf5d56e87dcd94ed8a2cb6e1754cb8e9de6d63ae33a6e5a1340c2a49f394R182